tests/PythonTest.sh

```diff
@@ -23,11 +23,18 @@ runtime_library_dir=${test_dir}/../python
 
 # Emit Python code for the example schema in the test dir:
 ${test_dir}/../flatc -p -o ${gen_code_path} -I include_test monster_test.fbs --gen-object-api
+${test_dir}/../flatc -p -o ${gen_code_path} -I include_test monster_test.fbs --gen-object-api --python-enum --python-version 3
```

Comment:
if i remember correctly only the common flag combinations are tested so i would not add it here, it is a bit a trade of between test speed and  completeness but with the amount of flags testing all combinations is a time consuming task